recipes/azure-keyvault-securitydomain/recipe.yaml

```diff
@@ -0,0 +1,47 @@
+schema_version: 1
+
+context:
+  name: azure-keyvault-securitydomain
+  version: 1.0.0b1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/azure_keyvault_securitydomain-${{ version }}.tar.gz
+  sha256: 3291a191e778a947e4b28ed01327892a93aedcf8e0a0dd674cf116cb11043776
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - isodate >=0.6.1
+    - azure-core >=1.31.0
+    - typing_extensions >=4.6.0
+
+tests:
+  - python:
+      imports:
+        - azure.keyvault.securitydomain
+        - azure.keyvault.securitydomain._internal
```

Comment:
This is prone to break and probably not useful.
```suggestion
```

Reply:
Agreed. Makes you wonder how `grayskull` decides which imports to add tests for.